examples/randomized-benchmark/DRAG-optimization/configuration.py

```diff
@@ -1,40 +1,83 @@
 import numpy as np
 
-gauss_len = 100
-gaussian_amp = 0.2
+gauss_len = 16  # pulse length in ns
+gaussian_amp = 0.2  # pulse amplitude in volts
 
+# Definition of pulses follow Chen et al. PRL, 116, 020501 (2016)
 
-def gauss(amplitude, mu, sigma, delf, length):
+
+# I-quadrature component for gaussian shaped pulses
```

Comment:
If you're at it, put the comments in the docstrings!
Actually, it would be great to add these to qualang_tools, and then just import it!!!!

But we can do it in 2 stages, first update this file to the github, then prepare the qualang_tools, then update this file again. Whatever you prefer.